### ISSUE TITLE
Input and output in pandas DataFrame are inverted in storage_invest example

### ISSUE THREAD

**Issue by uvchik** [oemof/oemof#190]
In the example the output of the natural gas power plant is summed up. The input is requested in line 144 but the output is returned. Which is good, because the output is wanted but it is confusing that one get the input by typing output and the output by typing input. Something seems to be inverted.

The efficiency is 0.58, which is exactly the difference between the input and the output but it should be the other way round. Would be great to fix it soon, because it has a big impact on the results if people are not aware of the this bug.
```
pp_gas_output_sum: 1.943970e+08
pp_gas_input_sum:  1.127503e+08
```

**Comment by simnh** [comment]
If you are talking about this: 

``` pp_gas = myresults.slice_by(obj_label='pp_gas', type='input',
                                date_from='2012-01-01 00:00:00',
                                date_to='2012-12-31 23:00:00')```

If i am correct, this will return the input into the bus (which is omitted, i do not know why) from by the object pp_gas so this will return electricity from gas turbine. 

Note that the `slice_by` method is from the bus perspective. Actually if there where another bus with the input of the object `pp_gas` in the call above, this would be included in the sum as well....

**Comment by simnh** [comment]
So I guess everything should work as it is supposed to work?

**Comment by ckaldemeyer** [comment]
> So I guess everything should work as it is supposed to work?

In my nodes_from_csv Branch everything seems to work as expected, too and I did some other fixes I don't remember now some weeks ago. I guess @simnh merged them into the release branch.

@uvchik : Could you check it again?

**Comment by uvchik** [comment]
> So I guess everything should work as it is supposed to work?

@simnh Kind of. It might not be a bug, but it definitely does not work as expected. The label is `pp_gas` and this implies, that `input` and `output`are referred to `pp_gas`. For me it is way from being intuitive if `obj_label='pp_gas', type='input'` means output from `pp_gas`. I would expect the following.

```python
pp_gas_input = myresults.slice_by(obj_label='pp_gas', type='input',
                                  date_from='2012-01-01 00:00:00',
                                  date_to='2012-12-31 23:00:00')
pp_gas_output = myresults.slice_by(obj_label='pp_gas', type='output',
                                   date_from='2012-01-01 00:00:00',
                                   date_to='2012-12-31 23:00:00')
```

Sorry. I do not understand the concept behind it and I might not be the only one. I guess it was an easier way to create the DataFrame. Could we at least rename the attribute `type` to make it more clear that this is the direction from the view of the bus. Or rename the labels from `input` to `inbus` and `output` to `frombus`.

@ckaldemeyer : I do not talk about `nodes_from_csv ` but about the `storage_invest` example. By the way, the `nodes_from_csv ` example does not work out of the box, but we should not discuss it here. Cool approach, anyway.

**Comment by uvchik** [comment]
Reading my comment again, I really like the idea of renaming `input` to `tobus` and `output` to `frombus`, because it works for buses and nodes.

**Comment by simnh** [comment]
I agree that we should try to avoid the counter-intuitive behavior of the method. Your proposals are ok for me. Who can do this, maybe @s3pp?

**Comment by ckaldemeyer** [comment]
> Sorry. I do not understand the concept behind it and I might not be the only one. I guess it was an easier way to create the DataFrame. Could we at least rename the attribute type to make it more clear that this is the direction from the view of the bus. Or rename the labels from input to tobus and output to frombus.

I agree, too. That necessary "maintenance work" which seems to be not possible at the moment (due to a lack of time) is also why I wanted the outputlib as lean as possible in the beginning.

> @ckaldemeyer : I do not talk about nodes_from_csv but about the storage_invest example. By the way, the nodes_from_csv example does not work out of the box, but we should not discuss it here. Cool approach, anyway.

:+1: Yep, it has to be prepared and is as experimental as the CSV-Reader. I also have some suggestions and ideas for the future. Let's discuss it in the PR.

**Comment by uvchik** [comment]
> I agree, too. That necessary "maintenance work" which seems to be not possible at the moment (due to a lack of time) is also why I wanted the outputlib as lean as possible in the beginning.

Yes, maintenance is annoying, me and @simnh already know that full well... . But as creating a DataFrame is one of the core functions of this library a leaner outputlib would be empty and do nothing.

**Comment by uvchik** [comment]
fixed by #191
